ui/components/multichain/token-list-item/token-list-item.test.js

```diff
@@ -39,7 +39,8 @@ describe('TokenListItem', () => {
   it('should render correctly', () => {
     const store = configureMockStore()(state);
     const { getByTestId, container } = renderWithProvider(
-      <TokenListItem />,
+      // eslint-disable-next-line no-empty-function
+      <TokenListItem onClick={() => {}} />,
```

Comment:
what's the goal of adding an empty function here ?

Reply:
This PR makes `onClick` optional for `TokenListItem` since we want to re-use the component, but unlike the home page, clicking should not take you anywhere.  Passing this keeps the test rendering what it used to.